kuma/redirects/redirects.py

```diff
@@ -0,0 +1,51 @@
+from redirect_urls import redirect
+
+# Redirects/rewrites/aliases migrated from SCL3 httpd config
+redirectpatterns = [
+    # RewriteRule ^/media/(redesign/)?css/(.*)-min.css$
+    # /static/build/styles/$2.css [L,R=301]
+    redirect(r'^media/(?:redesign/)?css/(?P<doc>.*)-min.css$',
```

Comment:
Might want the `prepend_locale=False` arg on some/most of these. Probably works without, but more regex than you need.